.github/workflows/porter-install-check.yml

```diff
@@ -25,8 +25,7 @@ jobs:
       shell: powershell
   macos:
     runs-on:
-      - self-hosted
-      - macOS-latest
+      - macos-latest
```

Comment:
Did we want to do this on Windows? I know that @ludfjig  was having issues with this at times. If it's easier not to, we don't have to.

Reply:
Nevermind, I see we already do this.